package.json

```diff
@@ -10,46 +10,48 @@
     "e2e": "ng e2e",
     "build_lib": "ng build ngx-powerbi",
     "npm_pack": "cd dist/ngx-powerbi && npm pack",
-    "package": "npm run build_lib && npm run npm_pack"
+    "package": "npm run build_lib && npm run npm_pack",
+    "build_lib_prod": "ng build ngx-powerbi --prod",
+    "prepublish_lib": "npm run build_lib_prod",
+    "publish_lib": "npm publish dist/ngx-powerbi"
   },
   "private": true,
   "dependencies": {
-    "@angular-devkit/build-ng-packagr": "^0.10.5",
-    "@angular/animations": "^7.0.0",
-    "@angular/common": "^7.0.0",
-    "@angular/compiler": "^7.0.0",
-    "@angular/core": "^7.0.0",
-    "@angular/forms": "^7.0.0",
-    "@angular/http": "^7.0.0",
-    "@angular/platform-browser": "^7.0.0",
-    "@angular/platform-browser-dynamic": "^7.0.0",
-    "@angular/router": "^7.0.0",
-    "core-js": "^2.5.4",
-    "ng-packagr": "^4.4.0",
+    "@angular/animations": "~9.0.1",
+    "@angular/common": "~9.0.1",
+    "@angular/compiler": "~9.0.1",
+    "@angular/core": "~9.0.1",
+    "@angular/forms": "~9.0.1",
+    "@angular/platform-browser": "~9.0.1",
+    "@angular/platform-browser-dynamic": "~9.0.1",
+    "@angular/router": "~9.0.1",
     "powerbi-client": "^2.6.5",
-    "rxjs": "~6.3.3",
-    "zone.js": "~0.8.26"
+    "rxjs": "~6.5.4",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.10.0",
-    "@angular/cli": "~7.0.5",
-    "@angular/compiler-cli": "^7.0.0",
-    "@angular/language-service": "^7.0.0",
-    "@types/jasmine": "~2.8.8",
+    "@angular-devkit/build-angular": "~0.900.2",
+    "@angular-devkit/build-ng-packagr": "~0.900.6",
+    "@angular/cli": "~9.0.2",
+    "@angular/compiler-cli": "~9.0.1",
+    "@angular/language-service": "~9.0.1",
```

Comment:
can you update these to the latest version i.e. 9.1.2?

Reply:
I'll do.

Reply:
Done.